pydoctor/model.py

```diff
@@ -427,11 +432,17 @@ class Class(CanContainImportsDocumentable):
     parent: CanContainImportsDocumentable
     bases: List[str]
     baseobjects: List[Optional['Class']]
-    decorators: Sequence[Tuple[str, Optional[Sequence[str]]]]
+    decorators: Sequence[Tuple[str, Optional[Sequence[ast.expr]]]]
+    raw_decorators: Sequence[ast.expr]
```

Comment:
Yes: while it's not used in pydoctor itself, it is used by Twisted's pydoctor plugin.

Reply:
I'll add a comment. Making it private wouldn't be worth the effort of syncing that change with Twisted, in my opinion, since it won't be part of a long term solution.